packages/ckeditor5-upload/src/uploadconfig.ts

```diff
@@ -44,29 +44,37 @@ export interface SimpleUploadConfig {
 	uploadUrl: string;
 
 	/**
-	 * An object that defines additional [headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers) sent with
-	 * the request to the server during the upload. This is the right place to implement security mechanisms like
+	 * Headers sent with the request to the server during the upload. This is the right place to implement security mechanisms like
```

Comment:
What is the purpose of removing the link from docs?

Reply:
It removed `An object that defines`, as it's no longer an object.

Reply:
Fixed https://github.com/ckeditor/ckeditor5/pull/17711/commits/5b2d2704b073eecfb96451a610aa85f5ae4cc782